deploy-cc/rbac-csi-azurefile-controller.yaml

```diff
@@ -180,6 +180,15 @@ rules:
     verbs: ["get", "create"]
 
 ---
+kind: ClusterRole
```

Comment:
Are these RBAC changes required because of `getRuntimeClassForPod()`?

Reply:
yes